### PR TITLE
Create Server App

### PR DESCRIPTION
**Pull request by ericmdai**
Implement a flask server application for airdrop

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    parameter). If no timestamp is provided, the current time is used instead.
+
+    Returns:
+        flask.wrappers.Response: Response object containing a json
+            representation of the queried state/setting.
+    """
+    if table not in tables:
+        return Response('<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n\
+            <title>404 Not Found</title>\n<h1>Not Found</h1>\n\
+            <p>The requested URL was not found on the server. \
```
Comment (RamV13) [inline app.py]: I think it should say the requested parameter was not found (i.e. state or setting not the URL)

File: app.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            If you entered the URL manually please check your spelling and try again.</p>\n',
+                        status=404, mimetype='text/html')  # Default flask 404
+
+    try:
+        timestamp = long(time.time() * 10**3) if request.args.get('timestamp') is None \
+            else long(request.args.get('timestamp'))
+    except:
```
Comment (RamV13) [inline app.py]: You should include the actual exception(s) you are catching in order to prevent masking other errors that may occur

File: app.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        return bad_request('Invalid timestamp query parameter')
+
+    try:
+        row = tables[table].query.filter(tables[table].timestamp <= timestamp).order_by(
+            tables[table].timestamp.desc()).first()
+    except:
```
Comment (RamV13) [inline app.py]: Same here

File: app.py (hunk 2 above)
Comment (ericmdai) [inline app.py]: Added to next PR.

File: app.py (hunk 3 above)
Comment (ericmdai) [inline app.py]: Same here.

File: setting.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    def __init__(self, timestamp=None, target=GPS(0, 0), threshold=0, armed=False, override=False):
+        """
+        Constructs a Setting object from the given arguments.
+
+        Args:
+            timestamp (Optional[long]): Timestamp of the setting in
+                milliseconds. If no timestamp provided, argument defaults to
+                None and is later set to the current time.
+        """
+        if timestamp is None:
```
Comment (RamV13) [inline setting.py]: I don't think you need this if statement and reassign

File: setting.py [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        return self.overide
+
+    def json(self):
+        """
+        Returns:
+            dict: A dictionary representation of the given setting to be used
+                for generating a JSON body.
+        """
+        return {
+            'timestamp': self.timestamp,
```
Comment (RamV13) [inline setting.py]: Update Apiary for this

**Comment by RamV13** [comment]
+1

**Comment by yogishadixit** [comment]
+1

**Pull request merged**